Examples/Swift/CalloutDelegateUsageExample.swift

```diff
@@ -60,6 +60,10 @@ class CalloutDelegateUsageExample_Swift: UIViewController, MGLMapViewDelegate {
         // Hide the callout view.
         mapView.deselectAnnotation(annotation, animated: false)
         
-        UIAlertView(title: annotation.title!!, message: "A lovely (if touristy) place.", delegate: nil, cancelButtonTitle: nil, otherButtonTitles: "OK").show()
+        // Create an alert containing the annotation's details
+        let alert = UIAlertController(title: annotation.title!!, message: "A lovely (if touristy) place.", preferredStyle: UIAlertControllerStyle.alert)
+        alert.addAction(UIAlertAction(title: "Ok", style: UIAlertActionStyle.default, handler: nil))
```

Comment:
- Capitalize both letters in “OK”.
- I think you can omit `UIAlertActionStyle`.